djangoproject/scss/_style.scss

```
@@ -2523,10 +2523,28 @@ table.foundation td {
     padding: 0 5px;
 }
 
-table.docutils td,
-table.docutils th {
-    border-bottom: 1px solid var(--hairline-color);
-}
+table.docutils {
+    border-collapse: collapse;
+    margin-bottom: 10.5em;
+    box-shadow: 0 1px 3px rgba(0,0,0,0.1);
+  }
+  table.docutils td, table.docutils th {
+    border-bottom: 1px solid #ccc;
```

Can we use a theme color (see `_utils.scss` in the same folder) instead of hardcoding `#ccc`?
```suggestion
    border-bottom: 1px solid $gray-medium;
```